s3-trigger-stack.yaml

```diff
@@ -155,7 +178,7 @@ Resources:
                           .format(','.join(bucket_where_trigger_exists))
                           )
 
-                      cfnresponse.send(event, context, cfnresponse.SUCCESS, response)
+                  cfnresponse.send(event, context, cfnresponse.SUCCESS, response)
```

Comment:
Why this change?

Reply:
it was same code in both if and else, so removed from both and keep it outside of if and else